run_experiment.py

```diff
@@ -99,12 +105,15 @@ def parse_arguments():
     parser = argparse.ArgumentParser(description="The primary script for running experiments on locally saved models.",
```

Comment:
I would say to visualize. Also I think the whole script should actually be called `main.py` and put in `viz`directory. + no link but I would not have called a file `viz/viz_helpers.py`: if it's in `viz` then it can just be called `helpers`. Both these points are personal preference feel free to do what you prefer (the only one I have a strong feeling bout is replace `experimets` with `plots` / `viz` because when I think about experiments I think about training or evaluating also )

Reply:
I like the idea of changing the name of `viz/viz_helpers.py`. I am more in favour of keeping the main script for the viz in the main directory though, but I also don't feel strongly about it.